mne/coreg.py

```diff
@@ -77,9 +77,8 @@ def create_default_subject(mne_root=None, fs_home=None, update=False,
 
     Parameters
     ----------
-    mne_root : None | str
-        The mne root directory (only needed if MNE_ROOT is not specified as
-        environment variable).
+    mne_root : None
+        This argument is not used anymore.
```

Comment:
Ideally we'd deprecate and remove it, but it's really hard to do that without breaking people's pipelines if it's the first argument. @agramfort any suggestions?